ibis/backends/base/sqlglot/__init__.py

```diff
@@ -62,12 +62,53 @@ def __getitem__(self, key: str) -> sg.exp.Column:
         return sg.column(key)
 
 
+class IdentiferGen:
+    __slots__ = ()
+
+    def __getattr__(self, name: str) -> sg.exp.Identifier:
+        return sg.to_identifier(name)
+
+    def __getitem__(self, key: str) -> sg.exp.Identifier:
+        return sg.to_identifier(key)
+
+
+def dot(first: str, second: str, *rest: str) -> sg.exp.Dot:
+    """Create a dot expression."""
+    return reduce(
+        lambda left, right: sg.exp.Dot(this=left, expression=right),
+        (
+            sg.to_identifier(first),
+            sg.to_identifier(second),
+            *map(sg.to_identifier, rest),
+        ),
+    )
+
+
+def paren(expr):
+    """Wrap a sqlglot expression in parentheses."""
+    return sg.exp.Paren(this=expr)
+
+
+def parenthesize(op, arg):
+    import ibis.expr.operations as ops
+
+    if isinstance(op, (ops.Binary, ops.Unary)):
+        return paren(arg)
+    # function calls don't need parens
+    return arg
+
+
+def lit(val):
```

Comment:
I will clean this up, we don't actually need it: sqlglot has a `convert` function that does essentially the same thing as this function.

Reply:
Done!